pkg/agent/route/route_linux.go

```diff
@@ -572,7 +576,17 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet, localAntreaFl
 			"-j", iptables.SNATTarget, "--to", snatIP.String(),
 		}...)
 	}
-
+	// The masqueraded multicast traffic will become unicast so we
+	// stop traversing this antreaPostRoutingChain for multicast traffic.
```

Comment:
why the traffic would become unicast after masqueraded? It only changes source IP, right?

Reply:
Yes, it changes source IP. However, only one interface of outbound interfaces can capture multicast traffic after masquerade. I found similar description here  https://bugzilla.kernel.org/show_bug.cgi?id=204067.